tests/scripts/all.sh

```diff
@@ -669,24 +686,38 @@ component_build_default_make_gcc_and_cxx () {
     make TEST_CPP=1
 }
 
-component_test_use_psa_crypto_full_cmake_asan() {
-    # MBEDTLS_USE_PSA_CRYPTO: run the same set of tests as basic-build-test.sh
+component_test_no_use_psa_crypto_full_cmake_asan() {
+    # full minus MBEDTLS_USE_PSA_CRYPTO: run the same set of tests as basic-build-test.sh
     msg "build: cmake, full config + MBEDTLS_USE_PSA_CRYPTO, ASan"
     scripts/config.pl full
     scripts/config.pl unset MBEDTLS_MEMORY_BACKTRACE # too slow for tests
-    scripts/config.pl unset MBEDTLS_ECP_RESTARTABLE  # restartable ECC not supported through PSA
+    scripts/config.pl set MBEDTLS_ECP_RESTARTABLE  # not using PSA, so enable restartable ECC
     scripts/config.pl set MBEDTLS_PSA_CRYPTO_C
```

Comment:
The merge is correct, but why is `MBEDTLS_PSA_CRYPTO_C` set here in mbedtls?

Reply:
The test used to be a "cmake, full config + MBEDTLS_USE_PSA_CRYPTO, ASan" as the message says, but is actually a "full minus MBEDTLS_USE_PSA_CRYPTO: run the same set of tests as basic-build-test.sh" as the comment says. I believe the `scripts/config.pl set MBEDTLS_PSA_CRYPTO_C` line is an artifact leftover from the test's previous purpose, same with how the message is incorrect now. This should be fixed separate from this merge.

Reply:
Fixing via https://github.com/ARMmbed/mbedtls/pull/2771